sdktests/common_tests_events_contexts.go

```diff
@@ -252,7 +252,7 @@ func (c CommonEventTests) EventContexts(t *ldtest.T) {
 					eventMatchers := []m.Matcher{debugEventMatcher, IsSummaryEvent()}
 					if c.isClientSide {
 						eventMatchers = append(eventMatchers, IsIdentifyEvent())
-					} else if !c.isPHP {
+					} else {
```

Comment:
The condition here was redundant because we already verified it's not PHP in the enclosing block.